test/rekt/resources/inmemorychannel/inmemorychannel.go

```diff
@@ -0,0 +1,79 @@
+/*
+Copyright 2021 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package inmemorychannel
```

Comment:
No need to do any of this. Set InMemoryChannel from the outside of the channelimpl tests and run them. https://github.com/knative/eventing/blob/main/test/rekt/resources/channel_impl/channel_impl.go#L46-L49